src/main/java/me/crespel/karaplan/service/PlaylistService.java

```diff
@@ -17,6 +17,8 @@
 	Optional<Playlist> findById(Long id);
 
 	Optional<Playlist> findById(Long id, boolean includeSongs);
+	
+	Playlist createPlaylist(String name, String username, boolean restricted);

```

Comment:
Prefer passing the User object instead of its name (same remark for other occurrences)